modules/brightcomSSPBidAdapter.js

```diff
@@ -0,0 +1,326 @@
+import {
+  getBidIdParameter,
+  isArray,
+  getWindowTop,
+  getUniqueIdentifierStr,
+  deepSetValue,
+  logError,
+  logWarn,
+  createTrackPixelHtml,
+  getWindowSelf,
+  isFn,
+  isPlainObject,
+} from '../src/utils.js';
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER} from '../src/mediaTypes.js';
+import {config} from '../src/config.js';
+import {ajax} from '../src/ajax.js';
+
+const BIDDER_CODE = 'bcmssp';
+const URL = 'https://rt.marphezis.com/hb';
+const TRACK_EVENT_URL = 'https://rt.marphezis.com/prebid'
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+  isBidRequestValid,
+  buildRequests,
+  interpretResponse,
+  onBidderError,
+  onTimeout,
+  onBidWon,
+  getUserSyncs,
+};
```

Comment:
Just want to make sure that the absence of gvlid is intentional but was meant to be added to the other bidder file?

Reply:
It's a mistake, I added the gvlid in the spec object. Thank you!